src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+import java.util.Random;
+public class FigureSupplier {
+    private static final Random random = new Random();
+
+    public static Figure getRandomFigure() {
+        String color = ColorSupplier.getRandomColor();
+
+        switch (random.nextInt(5)) {
+            case 0:
+                double side = random.nextDouble() * 10 + 1;
+                return new Square(side, color);

```

Comment:
Its better to create private method in this class which will create new Square
Same for all figures